torch/csrc/jit/script/compiler.cpp

```diff
@@ -1422,6 +1422,50 @@ struct to_ir {
     emitLoopCommon(range, body, assigner, {}, max_trip_count_val);
   }
 
+  void emitForInTensorLoop(const For& stmt, Value* tensorArg) {
+    auto targets = stmt.targets();
+    auto target = Var(targets[0]).name();
+    auto itrs = stmt.itrs();
+    auto body = stmt.body();
+    auto& range = stmt.range();
+
+    auto sizes_tuple = emitBuiltinCall(
+        range,
+        *graph,
+        aten::size,
+        c10::nullopt,
+        {tensorArg},
+        {},
+        /*required=*/true);
+
+    auto outermost_dim_index =
+        graph->insertConstant(0, nullptr, range);
```

Comment:
Why is the `result_type` null? Shouldn't it be `IntType::get()` or something?

Reply:
fixed.